applications/FluidDynamicsApplication/custom_utilities/mass_conservation_utility.h

```diff
@@ -254,30 +301,32 @@ class KRATOS_API(FLUID_DYNAMICS_APPLICATION) MassConservationCheckProcess : publ
      *
      * @param deltaDist Distance for the shift ( negative = more "water", positive = more "air")
      */
-    void ShiftDistanceField( double deltaDist );
+    void ShiftDistanceField( const double DeltaDist );
 
     /**
      * @brief Generating a 2D triangle of type Triangle2D3 out of a Triangle3D3 geometry
      * A rotation to a position parallel to the x,y plane is performed.
      * @param rGeom Original triangle geometry
      * @return Triangle2D3<Node<3>>::Pointer Shared pointer to the resulting triangle of type Triangle2D3
      */
-    Triangle2D3<Node<3>>::Pointer GenerateAuxTriangle( const Geometry<Node<3> >& rGeom );
+    Triangle2D3<Node<3>>::Pointer GenerateAuxTriangle( const Geometry<Node<3>>& rGeom );
 
     /**
      * @brief Function to generate an auxiliary line segment that covers only the negative part of the original geometry
      *
      * @param rGeom Reference to original geometry
-     * @param distance Distance of the initial boundary nodes
-     * @param p_aux_line Resulting line segment (output)
-     * @param aux_velocity1 Velocity at the first node of the new line segment(output)
-     * @param aux_velocity2 Velocity at the second node of the new line segment (output)
+     * @param rDistance Distance of the initial boundary nodes
+     * @param rpAuxLine Resulting line segment (output)
+     * @param rAuxVelocity1 Velocity at the first node of the new line segment(output)
+     * @param rAuxVelocity2 Velocity at the second node of the new line segment (output)
      */
     void GenerateAuxLine(   const Geometry<Node<3> >& rGeom,
```

Comment:
```suggestion
    void GenerateAuxLine(
        const GeometryType& rGeom,
```
There is a `typedef` above.

Reply:
done